BridgeAppSDK/Resources/ProfileItems.json

```diff
@@ -0,0 +1,46 @@
+{
+    "items":    [
+                 {
+                 "profileKey": "participantFirstName",
+                 "classType": "StudyParticipantProfileItem",
+                 "sourceKey": "firstName",
+                 "itemType": "String"
+                 },
+                 {
+                 "profileKey": "participantLastName",
+                 "classType": "StudyParticipantProfileItem",
+                 "sourceKey": "lastName",
+                 "itemType": "String"
+                 },
+                 {
+                 "profileKey": "participantEmail",
```

Comment:
Can we call this "participantUsername" instead? The overloading on the term "email" is really confusing. :)